test/BDNPerfTests/ConfigFiles/CI_CONFIG_BDN_Benchmark_Lua.LuaScripts.json

```diff
@@ -0,0 +1,19 @@
+{
+    "configuration": "Release",
+    "framework": "net8.0",
+    "filter": "BDN.benchmark.Lua.*",
+
+    "expectedScript1_None_AllocatedValue_linux": 24,
```

Comment:
these names should just be of the minimal form `"None": 24` ... we need to make this easy to understand and maintain. hopefully the script will automatically find the right entry by looking for the Param name, so the script does not need to change every time we add a benchmark. we only have to add a single line into the json file, that's it.

Reply:
We need to control in this line: test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts' ]   

Because if you just set the test some pre defined prefix "BDN_Tests*", then we lose the aspect where each test is ran concurrently. If you run all our BDNs in one filter, it takes 45+ mins.  

This also allows us to decide what BDNs we want to run and just don't assume if it is in the code, then it is ran in the CI.

As for variable names in the config file. It is helps to be descriptive. From other comment, we can remove the linux and win parts and just have one.  We do lose flexibility to look at other values, but I guess if we want to do that, we can just change the code again.

Reply:
Changed it to: "expectedScript1_None": 24,   from:   "expectedScript1_None_AllocatedValue_linux": 24   so it is a closer to minimal form and still be descriptive enough